pkgs/by-name/la/lazysql/package.nix

```diff
@@ -0,0 +1,31 @@
+{ lib
+, buildGoModule
+, fetchFromGitHub
+, xorg
+}:
+
+buildGoModule rec {
+  pname = "lazysql";
+  version = "0.1.8";
+
+  src = fetchFromGitHub {
+    owner = "jorgerojas26";
+    repo = "lazysql";
+    rev = "v${version}";
+    hash = "sha256-yPf9/SM4uET/I8FsDU1le9JgxELu0DR9k7mv8PnBwvQ=";
+  };
+
+  vendorHash = "sha256-tgD6qoCVC1ox15VPJWVvhe4yg3R81ktMuW2dsaU69rY=";
+
+  buildInputs = [
+    xorg.libX11
```

Comment:
Just a quick question about this - I saw the upstream issue about this. Do you know whether the package unconditionally needs libX11, or if it's possible to build without it on macos?

Reply:
@h7x4 I pulled out my Mac (M1) to test it, and it looks like it doesn't need it, just needs `Cocoa`. I can push those as changes if you'd like, making `libX11` depend on `isLinux` and `Cocoa` depend on `isDarwin`, if that sounds about right?

I don't have the devices to `x86_64-darwin` or `aarch64-linux`, but it compiles and runs on both `x86_64-linux` and `aarch64-darwin`.

Reply:
That sounds good to me. I think we could just rely on ofborg building it for those architectures, assuming that they work, and rather fix it later if anyone complains.

Reply:
Mac architecture should be added now @h7x4

Reply:
Cool. Just FYI, [the aarch64-darwin ofborg builders are having having a hard time keeping up](https://ofborg.org/prometheus/graph?g0.expr=ofborg_queue_builder_waiting%7Barch!%3D%22aarch64-darwin-lowprior%22%2Carch!%3D%22x86_64-darwin-lowprior%22%7D%20or%20ofborg_queue_evaluator_waiting&g0.tab=0&g0.stacked=0&g0.show_exemplars=0&g0.range_input=3d), so it might take some time for it to get built. But everything else looks fine, so feel free to ping me for merge when ofborg is done if I don't see it by then.

Reply:
Interesting, I'll be sure to let you know if I notice it being done. Really appreciate your time!

Reply:
@h7x4 Looks like everything passed! (If I understand `ofborg` at least lol)